File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Model/Workflow/WorkflowValidationRulesType.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VecompSoftware.DocSuiteWeb.Model.Workflow
{
    public enum WorkflowValidationRulesType : short
    {
        [Description("Esiste cartella di fascicolo")]
        IsExist,
        [Description("La cartella contiene almeno un file/inserto")]
        HasFile,
        [Description("La cartella contiene almeno una unità documentale")]
        HasDocumentUnit,
        [Description("La cartella contiene almeno un file/inserto firmato digitalmente")]
        HasSignedFile
    }
}

File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/Commons/WorkflowValidationRulesType.ts
enum WorkflowValidationRulesType {
    IsExist = "Esiste cartella di fascicolo",
    HasFile = "La cartella contiene almeno un file/inserto",
    HasDocumentUnit = "La cartella contiene almeno una unità documentale",
    HasSignedFile = "La cartella contiene almeno un file/inserto firmato digitalmente"
}

export = WorkflowValidationRulesType;